### PR TITLE
Adding .gifs for the helloworld page Issue: #11562 

### PR DESCRIPTION
**Pull request by zelahi**
Provide graphics for "Hello World" #11562.

I recorded and created .gifs for the hello world page as mentioned on issue: https://github.com/docker/docker/issues/11562#issuecomment-144246672

Summary:

I created a new folder in the following folder: /docker/docs/userguide
The images were embedded on the page: dockerizing.md.
All images are stored in:  /docker/docs/userguide/hello_world_images/

Signed-off-by: Zuhayr Elahi elahi.zuhayr@gmail.com

### CODE REVIEW

**Comment by zelahi** [comment]
I updated my commit write-up with the sign-off

**Comment by zelahi** [comment]
Signed-off-by: Zuhayr Elahi elahi.zuhayr@gmail.com

File: docs/userguide/dockerizing.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ So what happened to our container after that? Well Docker containers
 only run as long as the command you specify is active. Here, as soon as
 `Hello world` was echoed, the container stopped.
 
+  <img src="/userguide/hello_world_images/helloworld.gif">
```
Comment (thaJeztah) [inline docs/userguide/dockerizing.md]: Could you change the images to use standard Markdown notation? i.e.;

```Markdown
![hello-world](/userguide/hello_world_images/helloworld.gif)
```

After changing, it's good practice to check if it works correctly, by running `make docs` from the root of your repository, as explained here: https://github.com/docker/docker/blob/master/docs/README.md

**Comment by thaJeztah** [comment]
Thanks @zelahi it looks like one or more commits are not signed-off. In general we like all commits to be "squashed", so that a pull-request only consists of a single commit.

Our contribution guide explains how to rebase and squash your commits; https://docs.docker.com/project/work-issue/#pull-and-rebase-frequently but if you need assistance, please ask for help in the #docker-dev IRC channel.

I'd also like to have @moxiegirl to have a look here; Mary if you can have a quick look? AFAIK this is the first time we're using animated gifs on our docs, so I'd like to have your opinion just to be sure! :+1:

**Comment by zelahi** [comment]
okay, thanks. I guess that was my fault for not reading GordonTheTurtle's message properly.  Let me do that and modify this pull request

File: docs/userguide/dockerizing.md (hunk 1 above)
Comment (zelahi) [inline docs/userguide/dockerizing.md]: Thanks for the feedback.  I went ahead and fixed that and used markdown notation.  Hope it looks good.  =)

**Comment by moxiegirl** [comment]
@zelahi It looks like your rebase went wrong.  Here is a little cheat sheet:

image

**Comment by zelahi** [comment]
Hmm could I possibly close this pull request and open a new one.  I am wondering if that might help.  I want to recreate my dev environmeny and then submit this change again

**Comment by moxiegirl** [comment]
@zelahi Yes, that sounds like a plan.  I'll close this. Please reopen a new one with the   change.

**Pull request closed without merging**